code/game/objects/items/spear.dm

```diff
@@ -139,7 +139,9 @@
 	if(iseffect(AM)) //and no accidentally wasting your moment of glory on graffiti
 		return
 	user.say("[war_cry]", forced="spear warcry")
-	explosive.forceMove(AM)
+	if(ismob(AM))
+		user.forceMove(get_turf(AM))
+	explosive.forceMove(get_turf(AM))
```

Comment:
```suggestion
	explosive.forceMove(AM)
```
the grenade is supposed to be jabbed into your target, not the tile they're standing on

Reply:
this change is intentional athath

Reply:
why isn't it in the pr body, then?